ReactSkia/core_modules/RSkDeviceInfo.h

```diff
@@ -9,22 +9,34 @@
 #include "ReactCommon/TurboModule.h"
 #include "core_modules/RSkEventEmitter.h"
 #include "rns_shell/common/Window.h"
+#include "sdk/NotificationCenter.h"
 
 namespace facebook {
 namespace react {
 
-class RSkDeviceInfoModule : public RSkEventEmitter {
+class RSkDeviceInfoModule : public TurboModule {
     public:
       RSkDeviceInfoModule(
              const std::string &name,
              std::shared_ptr<CallInvoker> jsInvoker,
              Instance *bridgeInstance);
+      ~RSkDeviceInfoModule();
+
     private:
+
+        std::vector<std::string> events_ = {"didUpdateDimensions"};
+        std::string dimensionEventName_ = "dimensionEventNotification";
```

Comment:
Event name may not be required to be kept as class member. 
Can be used directly or defined as a macro